app/models/concerns/can_navigate_flexibly.rb

```diff
@@ -0,0 +1,5 @@
+# Used to flag forms as 'flexible' for navigational purposes
```

Comment:
Cracking idea! 🧀 Now I open the class and can see immediately whether its flexible or not. Added bonus; Codeclimate doesn't moan.

Only final request is that I found the comments in the deleted `BaseHelper` useful. Could they be brought into here to give a bit more of an explanation of what we mean by `CanNavigateFlexibly`?

Reply:
Good point! I've resurrected the comment.